Atomic/usersys.py

```diff
@@ -0,0 +1,11 @@
+from Atomic.util import output_json
+from Atomic.discovery import RegistryInspect
+#atomic-registry.usersys.redhat.com:5000/auth/baude_auth
```

Comment:
This whole file is just a test file.  Wasn't meant to be in the commit.  Removing...